cmd/ocm/delete/cmd.go

```diff
@@ -40,14 +40,32 @@ var args struct {
 }
 
 var Cmd = &cobra.Command{
-	Use:       "delete [flags] PATH",
+	Use:       "delete [flags] PATH|RESOURCE_ALIAS(with resource_id)",
 	Short:     "Send a DELETE request",
 	Long:      "Send a DELETE request to the given path.",
 	RunE:      run,
 	ValidArgs: urls.Resources(),
 }
 
+var arguements_desc_list = map[string]string{
+	"account": `# Delete an account with ACCOUNT_ID
+  ocm delete [--parameter "deleteAssociatedResources=true"] account ACCOUNT_ID`,
+	"subscription": `# Delete a subscription with SUBSCRIPTION_ID
+  ocm delete subscription SUBSCRIPTION_ID`,
+	"role_binding": `# Delete a role binding with ROLE_BINDING_ID
+  ocm delete role_binding ROLE_BINDING_ID`,
+	"sku_rule": `# Delete an SKU role with SKU_ROLE_ID
+  ocm delete sku_rule SKU_ROLE_ID`,
+	"cluster": `# Delete acluster with CLUSTER_ID
+  ocm delete [--parameter "best_effort=true"] [--parameter "deprovision=true"] [parameter "dry_run=true"] cluster CLUSTER_ID`,
```

Comment:
these `--parameters` are only specific to cluster?

Reply:
even I have limited information about this one